search/search_repository_whitebox_test.go

```diff
@@ -47,7 +47,7 @@ func TestSearchByText(t *testing.T) {
 	t.Parallel()
```

Comment:
I'm not sure it is a good idea to have parallel tests that access the database. I suggest to get rid of the `t.Parallel()` here.